zulip_bots/zulip_bots/lib.py

```diff
@@ -18,6 +18,8 @@
 
 from zulip import Client
 
+default_empty_message_response = "Oops. Your message was empty."
```

Comment:
Maybe we can plug that string directly in the function, since it only appears one time and your comment down below explains that it is used as a default empty message response.